server/player.lua

```diff
@@ -37,15 +37,14 @@ function Login(source, citizenid, newData)
         return false
     end
 
-    local license = GetPlayerIdentifierByType(source --[[@as string]], 'license2') or GetPlayerIdentifierByType(source --[[@as string]], 'license')
-    local userId = storage.fetchUserByIdentifier(license)
+    local license, license2 = GetPlayerIdentifierByType(source --[[@as string]], 'license'), GetPlayerIdentifierByType(source --[[@as string]], 'license2')
+    local userId = storage.fetchUserByIdentifier(license2 or license)
```

Comment:
Should we check license and license2 here as well?

Reply:
@mafewtm What do you think?

Reply:
whatevers better i guess

Reply:
Attempting to fetch by both now and also asserting that a user entity exists in the database.